client/web/compose/src/components/ModuleFields/Configurator/Select.vue

```diff
@@ -81,13 +81,12 @@
               </b-td>
 
               <b-td class="d-flex align-items-center justify-content-center">
```

Comment:
While you're here can you replace this class with `align-middle text-right` to match other lists.
You can also add `align-middle` to the first b-td to center it.